docs/spec/identities.md

```diff
@@ -88,7 +88,47 @@ sections.
 
 The `root` of a verified `Identity` is the stable identifier of the repository.
 
-> **TODO** Insert specification of URNs here
+### Radicle URNs
+
+Entities within the Radicle network are uniquely identified by a Radicle URN
+which fits the functional definition of a URN as defined in [rfc8141].
+
+Note: Radicle URNs are not prefixed by `urn:`, because they are not officially
+registered.
+
+This is what a typical Radicle URN looks like:
+```
+rad:git:hwd1yredksthny1hht3bkhtkxakuzfnjxd8dyk364prfkjxe4xpxsww3try
+```
+
+If we'd map it onto a [rfc8141] URN, then the Namespace Identifier (NID) part
+would be `rad` and the Namespace Specific String (NSS):
+`git:hwd1yredksthny1hht3bkhtkxakuzfnjxd8dyk364prfkjxe4xpxsww3try`.
+
+The NSS part for our `rad` NID can be further broken down into:
+
+```
+<PROTOCOL>:<ID>/<PATH>
+```
+Where `<PROTOCOL>` can be either `git` or `pijul`, `<ID>` is a 59 character long
```

Comment:
I wouldn't mention `pijul` here since it can't actually be `pijul` (yet) :smile: